src/sass/quote/_base.scss

```diff
@@ -28,6 +30,10 @@
       font-size: $ts-32 * 4;
     }
   }
+
+  .#{$prefix}-avatar {
+    margin-right: $spacing-md;
```

Comment:
Lets bump this down to `$spacing-sm`. Right now the avatar image feels too far away/disconnected from the quote text.